src/library/tracksettablemodel.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "library/basesqltablemodel.h"
+
+class TrackSetTableModel : public BaseSqlTableModel {
+    Q_OBJECT
+
+};
```

Comment:
Why does this class exist if it doesn't define any methods?

Reply:
because it's a WIP ;)

Reply:
Okay :smile: